wdl/transforms/cascades/src/test/cases/no_input_no_output_workflow.wdl

```diff
@@ -0,0 +1,27 @@
+version development
+
+workflow no_input_no_output {
+  call no_inputs as noi1
```

Comment:
If we're testing formatting stuff here, it might be useful to have cases with tabs and spaces, too.